repose-aggregator/src/docs/asciidoc/architecture/performance.adoc

```diff
@@ -48,6 +48,59 @@ As Gatling runs, it will pipe the performance data it captures to InfluxDB.
 Once Gatling execution completes, the cloud resources will be cleaned up, and the Ansible playbook will finish executing.
 At some point, Grafana will query InfluxDB to retrieve the performance data which will be made available in the form of one or more dashboards.
 
+==== Gatling
+
+Data will be transported from Gatling to InfluxDB using the Graphite line protocol.
+The data being transferred will include numerous statistics calculated for a distinct time.
+To get a better idea of the data in question, a sample has been provided which captures the number of request/response pairs sent at time `1492629621` as well as various statistics calculated from the amount of time taken to service those requests.
+Note that the sample data only captures a single second of the Gatling simulation.
+Much more data of the same form would be expected for every second that the Gatling simulation ran.
+
+....
+gatling.computerworld.getComputers.ok.count 2 1492629621
+gatling.computerworld.getComputers.ok.min 192 1492629621
+gatling.computerworld.getComputers.ok.max 219 1492629621
+gatling.computerworld.getComputers.ok.mean 205 1492629621
+gatling.computerworld.getComputers.ok.stdDev 13 1492629621
+gatling.computerworld.getComputers.ok.percentiles50 192 1492629621
+gatling.computerworld.getComputers.ok.percentiles75 219 1492629621
+gatling.computerworld.getComputers.ok.percentiles95 219 1492629621
+gatling.computerworld.getComputers.ok.percentiles99 219 1492629621
+gatling.computerworld.getComputers.ko.count 0 1492629621
+gatling.computerworld.getComputers.all.count 2 1492629621
+gatling.computerworld.getComputers.all.min 192 1492629621
+gatling.computerworld.getComputers.all.max 219 1492629621
+gatling.computerworld.getComputers.all.mean 205 1492629621
+gatling.computerworld.getComputers.all.stdDev 13 1492629621
+gatling.computerworld.getComputers.all.percentiles50 192 1492629621
+gatling.computerworld.getComputers.all.percentiles75 219 1492629621
+gatling.computerworld.getComputers.all.percentiles95 219 1492629621
+gatling.computerworld.getComputers.all.percentiles99 219 1492629621
+....
+
+==== InfluxDB
+
+InfluxDB will process the data it receives through its Graphite listener port by applying configured templates.
+These templates will translate the metric names from the Graphite line protocol into measurements, fields, tags, and values in the database.
+
+For example, given the sample data provided in the <<Gatling, Gatling section>>, the following template could be applied:
+
+....
+gatling.*.*.*.* measurement.simulation.request.status.field
+....
+
+Resulting in a database full of entries like the following:
+
+....
+name: gatling # <1>
+time                count max mean min percentiles50 percentiles75 percentiles95 percentiles99 request                  simulation    status stdDev
```

Comment:
tl;dr: A table is totally the way to go. Silly me for not seeing it sooner.

Regarding the other stuff...

**Scrollbar**
I think we may still end up needing to do this with a table. The rendered horizontal scrollbar should be indication enough that one can scroll the block. The only problem would be Mac users who have "Show scroll bars" set to hide them, but I think that's on them.

**Font Size**
I don't like the font size of our new docs in general, and I was convinced the font size of the fixed-width text was larger. I even put together some screenshots to prove it, but apparently it's not the case. In fact, it seems the line height (i.e. line spacing) of the fixed-width text might even be smaller. I'm guessing the bold look and fixed-width part of the text led me to believe the font size was larger.

In fact, while the font size in general is still larger than a lot of other docs I see, the screenshots (included below) also showed that apparently I really dislike the line height, too. This is, of course, totally out of scope of this, but I thought I would point out what I was going on about.

The sections are:
(first row) our old docs, Jenkins docs, Puppet firewall docs, Rackspace blog, our new docs
(second row) our new docs, our new docs fixed-width text

![font sizes](https://cloud.githubusercontent.com/assets/6766560/25436662/9822cdd8-2a59-11e7-96bd-a7819efc795f.png)